src/Files.App.Controls/ThemedIcon/ThemedIcon.Owner.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) 2024 Files Community
```

Comment:
This is essentially handling toggle state of the owning control, is that correct?

Reply:
@0x5bfa wanted to simplify the **ThemedIcon.cs** code, and so separated this into its own partial file.  It was in that file previously

Reply:
> This is essentially handling toggle state of the owning control, is that correct?

Yes. IsToggle change for ToggleButton and AppToggleButton class and IsEnabled change for Control class.

Reply:
Alright, good idea.